Apps/Common/src/AccessManagement/Administration/KeycloakUserAdmin.cs

```diff
@@ -152,27 +143,23 @@ public async Task<int> DeleteUser(string userId, string base64BearerToken)
             int returnCode = 0;
             Uri baseUri = new Uri(this.configuration.GetSection(KEYCLOAKADMIN).GetValue<string>(DELETEUSERURL));
 
-            using (HttpClient client = this.GethttpClient(baseUri, base64BearerToken))
+            using HttpClient client = this.GethttpClient(baseUri, base64BearerToken);
```

Comment:
the lines below are repeated for each method, lets create a private method instead that can be reused